images/patroni/entrypoint.sh

```diff
@@ -10,13 +10,26 @@ fi
 cat > /home/postgres/patroni.yml <<__EOF__
 bootstrap:
   dcs:
-    loop_wait: ${PATRONI_LOOP_WAIT}
-    ttl: ${PATRONI_TTL}
-    retry_timeout: ${PATRONI_RETRY_TIMEOUT}
-    master_start_timeout: ${PATRONI_MASTER_START_TIMEOUT}
-    maximum_lag_on_failover: ${PATRONI_MAXIMUM_LAG_ON_FAILOVER}
     postgresql:
       use_pg_rewind: true
+    parameters:
+      shared_buffers: '32GB'
+      max_connections: 300
+      max_worker_processes: 56
+      max_parallel_workers: 56
+      max_parallel_workers_per_gather: 28
+      work_mem: '1GB'
+      maintenance_work_mem: '1GB'
+      temp_file_limit: '20GB'
+      idle_in_transaction_session_timeout = '60min'
+      wal_buffers: '128MB'
+      max_wal_size: '8GB'
+      min_wal_size: '1GB'
+      checkpoint_completion_target: 0.9
+      default_statistics_target: 1000
+      effective_cache_size: '192GB'
+      effective_io_concurrency: 8
+      random_page_cost: 1.5
```

Comment:
Why hardcode all values instead of using env variable values that are provided from the Helm chart?